packages/ariakit/src/form/__examples__/form/test.tsx

```diff
@@ -69,6 +69,15 @@ test("fix error on change", async () => {
   expect(getError()).not.toBeInTheDocument();
 });
 
+test("reset form on reset", async () => {
+  render(<Example />);
+  await press.Tab();
+  await type("John");
+  await press.Tab();
```

Comment:
I think we're focusing on the Submit button now.
```suggestion
  await press.Tab();
  await press.Tab();
  expect(getByRole("button", { name: "Reset" }));
```